sources/mysql/infoschema.go

```diff
@@ -207,12 +207,17 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 			ignored.AutoIncrement = true
```

Comment:
We are still setting ignored.Default if we find a default column, we should remove that.

Reply:
To be done later.